core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -958,36 +958,82 @@ private static String toSql(RelNode root, SqlDialect dialect) {
    * <a href="https://issues.apache.org/jira/browse/CALCITE-3220">[CALCITE-3220]
    * HiveSqlDialect should transform the SQL-standard TRIM function to TRIM,
    * LTRIM or RTRIM</a>. */
-  @Test public void testHiveTrim() {
+  @Test public void testHiveAndBqTrim() {
```

Comment:
As you can see there is a comment about this test case which is hive related. So you can also add comments for BQ.

You can either add another comment for BQ, or split this test case to hive and BQ and then add comments seperately.

Reply:
@amaliujia I have remove this check as this is an unnessary check over here because in SqlTrimFunction class checkOperandTypes method perform the Operands check.
Removed this assert as the operands type check is already done in SqlTrimFunction class.

Reply:
Added the comment for the BQ test cases along with JIRA Id